### PR TITLE
DashoGames.PiozaGameLauncher version 0.4.0

### PR DESCRIPTION
**Pull request by Shieldowskyy**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162929)

### CODE REVIEW

**Comment by Shieldowskyy** [comment]
@microsoft-github-policy-service agree company="DashoGames"

File: manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DashoGames.PiozaGameLauncher
+PackageVersion: 0.4.0
+InstallerLocale: pl-PL
+InstallerType: exe
```
Comment (Trenly) [inline manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.installer.yaml:7]:
```suggestion
InstallerType: inno
```

File: manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+InstallerLocale: pl-PL
+InstallerType: exe
+ProductCode: '{88525005-EC45-4536-A18D-09E99B266E54}'
+Installers:
+- InstallerUrl: https://github.com/Shieldowskyy/PiozaGameLauncher/releases/download/v.0.4.0/PiozaGL.exe
+  Architecture: x64
+  InstallerSha256: c0ae7fe5ab61cda6005ba9e26e8abedda27ea1f080b969c43a27642c6d67c9c4
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
```
Comment (Trenly) [inline manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.installer.yaml:15]:
```suggestion
```
These should be the default for Inno based installers

File: manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DashoGames.PiozaGameLauncher
+PackageVersion: 0.4.0
+InstallerLocale: pl-PL
```
Comment (Trenly) [inline manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.installer.yaml:6]:
```suggestion
```
The installer is not specific to the `pl-PL` locale. If this is included, users of other locales may not be able to install the software

File: manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: DashoGames.PiozaGameLauncher
+PackageVersion: 0.4.0
+PackageLocale: en-US
+Publisher: DashoGames
+PackageName: Pioza Game Launcher
```
Comment (Trenly) [inline manifests/d/DashoGames/PiozaGameLauncher/0.4.0/DashoGames.PiozaGameLauncher.locale.en-US.yaml:8]:
```suggestion
PackageName: PiozaGameLauncher
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> 2024-07-15 09:10:30.315 [FAIL] Installer failed security check. Url: https://github.com/Shieldowskyy/PiozaGameLauncher/releases/download/v.0.4.0/PiozaGL.exe Result: 0x80004005

(Automated response - build 886.)

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Shieldowskyy,

It appears that this package has a couple of dependencies:
image
image

**Comment by stephengillie** [comment]
Verify with above dependencies:
image
- https://github.com/microsoft/winget-pkgs/issues/107639
- https://github.com/microsoft/winget-pkgs/issues/12906

image

**Pull request closed without merging**